node_client/pyXain/pyXain/pyXain.py

```diff
@@ -208,19 +210,29 @@ def submit_manuscript(self, author_id, paper_id, paper_directory):
         author_check = self.check_author(author_id)
         if not author_check['Success'] :
             output = {}
-            output['Succcess'] = False
+            output['Success'] = False
```

Comment:
Noooo was this what was going wrong in that test? Too many c's?

Reply:
Yes it was, there was a key error that lead me to it. -_-